docs/CONTRIBUTING.md

```diff
@@ -1,30 +1,60 @@
-Contributing
-============
+# Contributing
 
-DJA should be easy to contribute to.
+Django REST Framework JSON API (aka DJA)  should be easy to contribute to.
 If anything is unclear about how to contribute,
 please submit an issue on GitHub so that we can fix it!
 
-How
----
+Before writing any code, have a conversation on a GitHub issue to see
+if the proposed change makes sense for the project.
 
-Before writing any code,
-have a conversation on a GitHub issue
-to see if the proposed change makes sense
-for the project.
+## Setup development environment
 
-Fork DJA on [GitHub](https://github.com/django-json-api/django-rest-framework-json-api) and
-[submit a Pull Request](https://help.github.com/articles/creating-a-pull-request/)
-when you're ready.
+### Clone
 
-For maintainers
----------------
+To start developing on Django REST Framework JSON API you need to first clone the repository:
 
-To upload a release (using version 1.2.3 as the example):
+    git clone https://github.com/django-json-api/django-rest-framework-json-api.git
 
-```bash
-(venv)$ python setup.py sdist bdist_wheel
-(venv)$ twine upload dist/*
-(venv)$ git tag -a v1.2.3 -m 'Release 1.2.3'
-(venv)$ git push --tags
-```
+### Testing
+
+To run tests clone the repository, and then:
+
+     # Setup the virtual environment
```

Comment:
@sliverc my point of `tox -e devenv` is you can turn these several steps into one.